client/dashboard/data/me-purchases.ts

```diff
@@ -14,3 +14,14 @@ export async function fetchUserTransferredPurchases(): Promise< Purchase[] > {
 		apiVersion: '1.1',
 	} );
 }
+
+export async function setPurchaseAutoRenew(
+	purchaseId: number,
+	autoRenew: boolean
+): Promise< { success: boolean; upgrade: Purchase } > {
+	const action = autoRenew ? 'enable-auto-renew' : 'disable-auto-renew';
+	return wpcom.req.post( {
+		path: `/upgrades/${ purchaseId }/${ action }`,
```

Comment:
Hmmm this endpoint is not actually in `/me/purchases`... is this an anomaly, or are the more endpoints like this? Maybe we should put it in `data/upgrades.ts` instead 🤔